packaging/windows/WINDOWS_INSTALLER.md

```diff
@@ -96,11 +80,65 @@ After installation, open your browser and go to:
 http://localhost:19999
 ```
 
----
-
 ## License Information
 
 By using silent installation, you agree to:
 
 - [GPL-3 License](https://raw.githubusercontent.com/netdata/netdata/refs/heads/master/LICENSE) — Netdata Agent
-- [NCUL1 License](https://app.netdata.cloud/LICENSE.txt) — Netdata Web Interface
\ No newline at end of file
+- [NCUL1 License](https://app.netdata.cloud/LICENSE.txt) — Netdata Web Interface
+
+## Automatic Updates
+
+For users who want to keep their Windows agents automatically updated with the latest releases, you can set up automated updates.
+
+:::tip 
+
+**What You'll Learn**
+
+How to setup automatic Netdata updates on Windows nodes using PowerShell and Task Scheduler.
+
+:::
+
+### How to setup:
+
+**1. Create `C:\netdata` dir.**
+
+```powershell
+New-Item -Path "C:\netdata" -ItemType Directory -Force
+```
+
+**2. Create `C:\netdata\netdata-updater.ps1`**
+
+Create the file with this exact content:
+
+```powershell
+Invoke-WebRequest https://github.com/netdata/netdata-nightlies/releases/latest/download/netdata-x64.msi -OutFile C:\netdata\netdata-x64.msi
+msiexec /qn /i C:\netdata\netdata-x64.msi TOKEN="<CLAIM_TOKEN>" ROOMS="<ROOM_ID>" 
+```
+
+:::note 
+
+**Configuration Required**
+
+Replace `<CLAIM_TOKEN>` with your Netdata Cloud claim token and `<ROOM_ID>` with your room identifier.
+
+:::
+
+**3. Create an entry in `Task Scheduler`:**
+
+Configure the task with these specific settings:
+
+**General tab** - check:
+- `Run whether user is logged in or not`
+- `Run with highest privileges`
```

Comment:
The `highest privleges` on Microsoft implies `as Administrator`, probably we should adjust this sentence to follow Windows standard.

Reply:
This are exact instructions. You do "Create Task" and it has "Run with highest privileges", not "as Administrator".

Please undo, @kanelatechnical. 
 
<img width="929" alt="Screenshot 2025-05-27 at 20 07 05" src="https://github.com/user-attachments/assets/0d5f28be-ab7b-4149-8d11-97c6c3ef9a18" />